packages/beacon-node/src/eth1/eth1DepositDataTracker.ts

```diff
@@ -109,10 +116,21 @@ export class Eth1DepositDataTracker {
     }
   }
 
+  stopPollingEth1Data(): void {
```

Comment:
if this is not consumed in this PR, mark `TODO` in the method header so that it's not missed